.buildkite/scripts/install_tools.sh

```diff
@@ -43,6 +43,10 @@ with_dependencies
 config_git
 mage dumpVariables
 
+if [[ "$BUILDKITE_PIPELINE_SLUG" == "beats-xpack-metricbeat" && "${BUILDKITE_STEP_KEY}" == "extended-cloud-test" ]]; then
+  startCloudTestEnv "${MODULE_DIR}"
```

Comment:
I'm not sure whether this is too much obfuscation.

I think this script should be called in the `.buildkite/scripts/cloud_tests.sh`


What's the reason to use this approach?

Reply:
@v1v I've updated your suggestion a bit to avoid some failures, please take a look at [this](https://github.com/elastic/beats/pull/38081/commits/1057a948a9dfde823529f3fc6297c96795164c5c) and [this](https://github.com/elastic/beats/pull/38081/commits/84e8305daa6c7b268b5e25e5f76bfabc87d86213) commits.

Reply:
Thanks. I've reviewed and come up with some thoughts I was unaware of earlier. Thanks for going through the review together and understand this